pdns/dnsdist.cc

```diff
@@ -1170,21 +1177,23 @@ ProcessQueryResult processQuery(DNSQuestion& dq, ClientState& cs, LocalHolders&
     gettime(&now);
 
     if (!applyRulesToQuery(holders, dq, now)) {
-      return ProcessQueryResult::Drop;
+      return false;
```

Comment:
Consider leaving the original return value. The breaking up of the method doesn't require changing the return value which makes the code more readable IMHO.

Reply:
The function had 3 return states(Drop, SendAnswer, PassToBackend) which make sense with the original implementation. But i broke the function into 2 different functions each returning true or false reply to query - true or false and pass query to backend - true or false. I think since each function now return only true or false this means a Boolean variable is preferred.
But if you still think this after my argument I can revert it.